packages/grid/_modules_/grid/hooks/features/columns/useGridColumns.ts

```diff
@@ -89,7 +88,7 @@ function updateColumnsWidth(columns: GridColumns, viewportWidth: number): GridCo
 
 function hydrateColumns(
   columns: GridColumns,
-  columnTypes: GridColumnTypesRecord,
+  columnTypes: GridColumnTypesRecord = {},
```

Comment:
I don't think the prop can ever be undefined
We have a default prop.

Reply:
Reverted.